src/officehours_api/models.py

```diff
@@ -74,6 +73,8 @@ def get_users_with_emails(manager: models.Manager):
 
 class Queue(SafeDeleteModel):
     _safedelete_policy = SOFT_DELETE_CASCADE
+    deleted_by_cascade = None
```

Comment:
The `1.2.0` release of `django-safedelete` introduced a field/column called `deleted_by_cascade`. However, it didn't supply defaults. The docs recommend and an issue I found suggest either adding this line or writing a migration to provide data for existing records. Since this field wouldn't do much for us (we have three fields using `SafeDeleteModel`, and only `Queue` has a soft delete policy), I think it's fine to opt out of this behavior. I'm open to adding a migration if someone thinks that's better.